Orchestrator/README.md

```diff
@@ -14,7 +14,7 @@ Orchestrator is an LU solution optimized for conversational AI applications. It
 
 **Intent recognition**: You can use Orchestrator as an intent recognizer with [adaptive dialogs][6] to route user input to an appropriate skill or sub-component.
 
-**Entity extraction** is not yet supported.
```

Comment:
We got to be careful here. There were no agreements to light this up yet. It's ok that we do the foundation work, but this may be too much too early.

Reply:
change to "is currently experimental and not yet for production use."